kyo-data/shared/src/main/scala/kyo/internal/Environment.scala

```diff
@@ -0,0 +1,11 @@
+package kyo.internal
+
+object Environment:
+
+    def isDevelopment(): Boolean =
+        sys.props.get("kyo.development-mode.enable").map(_.toLowerCase) match
```

Comment:
development can also be enabled explicitly via the `kyo.development-mode.enable` system property